Remove Object Async. With Args. (#464)
minio/minio-dotnet: Docs/API.md

```cs
// 1. Remove list of objects in objectNames from the bucket bucketName.
try
{
    string bucketName = "mybucket"
    List<String> objectNames = new LinkedList<String>();
    objectNames.add("my-objectname1");
    objectNames.add("my-objectname2");
    objectNames.add("my-objectname3");
    RemoveObjectsAsync rmArgs = new RemoveObjectsAsync()
                                            .WithBucket(bucketName)
                                            .WithObjects(objectNames);
    IObservable<DeleteError> observable = await minio.RemoveObjectAsync(rmArgs);
    IDisposable subscription = observable.Subscribe(
        deleteError => Console.WriteLine("Object: {0}", deleteError.Key),
        ex => Console.WriteLine("OnError: {0}", ex),
        () =>
        {
            Console.WriteLine("Listed all delete errors for remove objects on  " + bucketName + "\n");
        });
}
catch (MinioException e)
{
    Console.WriteLine("Error: " + e);
}

> // 2. Remove list of objects' (only specific versions mentioned in Version ID list) from the bucket bucketName

Author: Changed.

|Param   | Type	  | Description  |
|:--- |:--- |:--- |

> | ``args``  | _RemoveObjectArgs_ | Arguments Object - name.  |

Reviewer: ```suggestion
| ``args``  | _RemoveObjectArgs_ | Arguments Object  |
```

Author: Changed.

```cs
// 1. Remove object myobject from the bucket mybucket.
try
{
    RemoveObjectArgs rmArgs = new RemoveObjectArgs()
                                        .WithBucket("mybucket")
                                        .WithObject("myobject");
    await minioClient.RemoveObjectAsync(args);
    Console.WriteLine("successfully removed mybucket/myobject");
}
catch (MinioException e)
{
    Console.WriteLine("Error: " + e);
}

// 2. Remove one version of object myobject with versionID from mybucket.
try
{
    RemoveObjectArgs rmArgs = new RemoveObjectArgs()
                                        .WithBucket("mybucket")
                                        .WithObject("myobject")
                                        .WithVersionId("versionId");
    await minioClient.RemoveObjectAsync(args);
    Console.WriteLine("successfully removed mybucket/myobject{versionId}");
}
catch (MinioException e)
{
    Console.WriteLine("Error: " + e);
}

```
<a name="removeObjects"></a>
### RemoveObjectsAsync(RemoveObjectsArgs args)

`Task<IObservable<DeleteError>> RemoveObjectsAsync(RemoveObjectsArgs args, CancellationToken cancellationToken = default(CancellationToken))`

> Removes a list of objects or a list of specified versionIDs of each object in the list.

Reviewer: ```suggestion
Removes a list of objects or object versions.
```

Author: Changed.

```cs
// 1. Remove list of objects in objectNames from the bucket bucketName.
try
{
    string bucketName = "mybucket"
    List<String> objectNames = new LinkedList<String>();
    objectNames.add("my-objectname1");
    objectNames.add("my-objectname2");
    objectNames.add("my-objectname3");
    RemoveObjectsAsync rmArgs = new RemoveObjectsAsync()
                                            .WithBucket(bucketName)
                                            .WithObjects(objectNames);
    IObservable<DeleteError> observable = await minio.RemoveObjectAsync(rmArgs);
    IDisposable subscription = observable.Subscribe(
        deleteError => Console.WriteLine("Object: {0}", deleteError.Key),
        ex => Console.WriteLine("OnError: {0}", ex),
        () =>
        {

>             Console.WriteLine("Listed all delete errors for remove objects on  " + bucketName + "\n");

Reviewer: ```suggestion
            Console.WriteLine("Removed objects from " + bucketName + "\n");
```

Author: Changed message.
